src/main/java/frc/robot/RobotContainer.java

```diff
@@ -7,16 +7,28 @@
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import frc.robot.subsystems.pivot.Pivot;
+import frc.robot.subsystems.pivot.Pivot.PivotGoal;
+import frc.robot.subsystems.pivot.PivotConstants;
+import frc.robot.subsystems.pivot.PivotKrakenHardware;
 
 public class RobotContainer {
+  private final Pivot kPivot =
```

Comment:
We wouldn't use the name "kPivot" because this isn't a constant.  We can call it just "pivot", or perhaps "mPivot" if you like the m == instance variable naming scheme.

Reply:
This still needs to be addressed